_Sidebar.md

```diff
@@ -58,6 +58,7 @@
       * [[End-to-end tests|End-to-End-Tests]]
       * [[Acceptance Tests|Acceptance-Tests]]
       * [[Lighthouse Tests|Lighthouse-Tests]]
+      * [[Partial CI Tests Structure|Partial-CI-Tests-Structure]]
```

Comment:
Maybe you could rename this to: "Determining which CI tests run for a PR"

Reply:
Done.